frontend/src/metabase/routes.jsx

```diff
@@ -241,7 +241,13 @@ export const getRoutes = store => (
           <Route path=":slug/:objectId" component={QueryBuilder} />
         </Route>
 
-        <Route path="/model/:slug/detail" component={ModelDetailPage} />
+        <Route path="/model/:slug/detail">
```

Comment:
Just realize `react-router` v3 doesn't even have `<Switch />` wonder if we should considering upgrade this some time soon to have a better and more modern API available?

Reply:
For sure, we'd need to investigate how to better approach this first